ApiReview/API.Polly.Core/NoDocs/Polly.CircuitBreaker/CircuitBreakerManualControl.cs

```diff
@@ -0,0 +1,17 @@
+// Assembly 'Polly.Core'
+
+using System;
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Polly.CircuitBreaker;
+
+public sealed class CircuitBreakerManualControl : IDisposable
+{
+    public Task IsolateAsync(CancellationToken cancellationToken = default(CancellationToken));
```

Comment:
When I manually isolate a circuit ahead of time (because my app configuration says to) I need to do this call because configuring the service collection isn't async: `manualControl.IsolateAsync().GetAwaiter().GetResult()`.

This made me wonder if we could simplify this scenario by adding a constructor that allows you to set whether it's open when you create it? Then I could just do something like `var manualControl = new CircuitBreakerManualControl(open: true)` instead which is much neater.

From https://github.com/App-vNext/Polly/issues/1365#issuecomment-1620553627

Reply:
This is a valid use-case and I have no problem with this change.

Reply:
Done.